React/my-app/src/redux/accounts/actions.js

```diff
@@ -0,0 +1,46 @@
+import { checkUser, newUser } from "../../service/accounts";
+
+export const actions = {
+  NEW_USER_SUCCESS: "NEW_USER_SUCCESS",
+  NEW_USER_FAILURE: "NEW_USER_FAILURE",
+  CHECK_USER_SUCCESS: "CHECK_USER_SUCCESS",
+  CHECK_USER_FAILURE: "CHECK_USER_FAILURE"
+};
+
+export const verifyUser = body => {
+  return async dispatch => {
+    try {
+      const response = await checkUser(body);
+      if (response.status >= 200 && response.status < 300) {
```

Comment:
what if request fail, but just fail, not crash!
So you never get success, but never get failure too.